go/coding_standard.md

```diff
@@ -31,18 +31,29 @@ Please add items here, or modify them as appropriate. We have git versioning, do
  * Many comments are not sentences, these should begin with a lower case letter and end without a period.
  * The first letter of sentences in comments are capitalized and ends with a period.
  
-## Git
-
- * Production repos never push to `master` or `develop` branches except the push-master.
- * `develop` should always pass tests.
- * Always start work with a "WIP: \*" (work-in-progress) pull-request, on a branch ideally named "feature/\*" or "hotfix/\*". You're encouraged to fork the repo and add it as a remote in .git/config (so you can push to your fork, but still use the same import paths to the original project).
- * When complete, pull requests should be opened to the `develop` branch.
- * Do not merge PRs that aren't final -- e.g. no code dead code or code that won't be used should be committed.
- * Squash and merge to keep the history clean.
- * Merging from `develop` to `master` happens during a release.
- * All branches should be in lowercase, underscores are okay in branch names
- * All pull requests should include any relevant additions to the `CHANGELOG.md`, If PR is to the develop branch changes
- can be included under an `unreleased changes` header at the top of the document.
+## Github
+
+ * Production branches
+   * Production repos never push to `master` or `develop` branches except the push-master.
+   * `develop` should always pass tests.
+   * `develop` branch should have an on going open PR to `master`, high level comments can be made here
+   * Merging from `develop` to `master` happens during a release.
+ * Feature branches
+   * should following the naming convention `yourname/branch-description`
```

Comment:
* should following the naming convention `feature/<branch-description>`

Assign it to the owner, instead of adding the owner name to the branch name.

Reply:
I am using `666-my-branch-description`, so anyone can find branch I am working on by looking out issue number, typing 666 and pressing `TAB` (autofilling)

Reply:
An optional idea: adding the issue number in the feature description makes it easier to track issue/PR pairs. You might be able to do it by referencing the PR in the issue or vice versa, so I'm not sure if it's useful or not. Adding an "ID" to the description (Anton = 666) seems weird for me but not disturbing, so I don't have anything against it.

Reply:
I like branches with owner names - easier to identify right off the bat and know who is responsible for what just by looking at the branch names

Reply:
I still don't like it, but in a "disagree and commit" fashion, let's do it. I don't see any harm, except to make sure to tell new hires that this is how we do it.